src/components/shared/ScrollToFieldError/index.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react';
+
+export const ScrollToFieldError = ({
+  scrollBehavior = { left: 0, behavior: 'smooth' },
+  useFormikContext,
```

Comment:
Why do you need to inject `useFormikContext`? Can you not simply use the hook?

Reply:
makes the tests much easier ...

Reply:
Oh, I understand, that is the problem with the hooks!

maybe using a default value to simplify the code in the client component?

Reply:
yes, I'm agree. I'm going to do that ... thanks